packages/command-palette/src/commands.tsx

```diff
@@ -260,6 +269,20 @@ export function useCommands() {
 				siteSelectorLabel: __( 'Select site to open dashboard', __i18n_text_domain__ ),
 				icon: dashboardIcon,
 			},
+			...( isEnabled( 'layout/dotcom-nav-redesign-v2' ) && {
```

Comment:
This won't work on the wp-admin app (so this command won't be available there), but I think this is ok for now since the workaround would require to expose the `wpcom_classic_early_release` site option to wp-admin app and have a duplicated command there, which is unnecessary logic for production (i.e. once we launch the nav redesign project).

Reply:
Could we create a follow-up PR so that we enable this command for wp-admin once nav redesign is released?

Reply:
Worth adding a comment with the explanation. 🙂

Reply:
> Worth adding a comment with the explanation. 🙂
Once nav redesign is released, we can enable this command on Calypso and wp-admin since no check is needed. Having just a comment won't be enough, we need a follow-up so that we remember.

Reply:
> Could we create a follow-up PR so that we enable this command for wp-admin once nav redesign is released?

My plan was to open a follow-up PR that removes this flag check once the nav redesign is launched. Is that what you asked?

Reply:
> My plan was to open a follow-up PR that removes this flag check once the nav redesign is launched. Is that what you asked?

Yes, could we link it here for future reference?